cmd/containerd-shim-runhcs-v1/service.go

```diff
@@ -12,7 +12,7 @@ import (
 	"time"
 
 	task "github.com/containerd/containerd/api/runtime/task/v2"
-	"github.com/containerd/errdefs"
+	"github.com/containerd/containerd/errdefs"
```

Comment:
Is this revert needed?

Reply:
Yes, this is only for use of the stable version of `ToGRPC` in the containerd 1.7 package. The `github.com/containerd/errdefs/pkg/errgrpc` package shouldn't be considered stable yet.

Reply:
Looks like these changes are (re-)introducing a transitive dependency on containerd v1.7 (both for containerd v2 - https://github.com/containerd/containerd/commit/2f24aa00a58de442713ea60e4206041e00dcf012#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R95 ) - and for nerdctl v2?